Makefile.am

```diff
@@ -40,6 +40,8 @@ opx_pas_service_SOURCES+= src/fuse/pas_fuse_fan.c src/fuse/pas_fuse_common.c src
 
 opx_pas_service_SOURCES += src/pas_comm_dev.c src/pas_host_system.c src/pas/pas_comm_dev_handler.c src/pas/pas_host_system_handler.c
 
+opx_pas_service_SOURCES += src/pas_log.c src/pas_media_properties_discovery.c src/pas_media_info_map.cpp src/pas_media_properties_utils.c
```

Comment:
Minor nitpick: Should just combine this with the above line using a `\` character rather than concatenating like this.